scripts/heat/ping_template.sh

```diff
@@ -43,3 +43,18 @@ cat >> /usr/bin/ping_forever_neighbour << EOF
 EOF
 chmod +x /usr/bin/ping_forever_neighbour
 /usr/bin/ping_forever_neighbour &
+
+cat >> /usr/bin/cinder_test << EOF
+  #!/bin/sh
+
+  sudo mkfs -t ext4 -L cinder_volume /dev/vdb
```

Comment:
@jsuchome the cirros image has only one volume by default: `/dev/vda`. All my tests attaching a new (`cinder`) volume show it has been assigned to `/dev/vdb`. I think for our tests is a good enough solution.

@Itxaka you are right, even though you can name the device to use for the volume attachment, the cirros image will shamelessly ignore it.